certified-connectors/Webhood URL Scan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,372 @@
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Get past scans, optionally filter by status",
+        "operationId": "getScans",
+        "parameters": [
+          {
+            "name": "status",
+            "in": "query",
+            "description": "Status values that need to be considered for filter",
+            "x-ms-summary": "Scan Status",
+            "type": "string",
+            "enum": [
+              "pending",
+              "done",
+              "running",
+              "error"
+            ],
+            "default": "done"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Scan"
+              }
+            }
+          },
+          "400": {
+            "description": "Something went wrong while processing your request"
+          },
+          "401": {
+            "description": "The request requires valid record authorization token to be set"
+          },
+          "403": {
+            "description": "Authentication or authorization failure"
+          }
+        }
+      }
+    },
+    "/beta/scans/{scanId}": {
+      "get": {
+        "security": [{"apiKey": []}],
+        "tags": [
+          "scanner"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Get scan by ID",
+        "description": "Returns a single scan",
+        "operationId": "getScanById",
+        "parameters": [
+          {
+            "name": "scanId",
+            "in": "path",
+            "description": "ID of scan to return",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
+              "$ref": "#/definitions/Scan"
+            }
+          },
+          "400": {
+            "description": "Invalid ID supplied"
+          },
+          "401": {
+            "description": "The request requires valid record authorization token to be set"
+          },
+          "403": {
+            "description": "Authentication or authorization failure"
+          },
+          "404": {
+            "description": "Scan not found"
+          }
+        }
+      }
+    },
+    "/beta/scans/{scanId}/screenshot": {
+      "get": {
+        "security": [{"apiKey": []}],
+        "tags": [
+          "scanner"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "image/png"
+        ],
+        "summary": "Get screenshot by scan ID",
+        "description": "Returns a screenshot of a successful scan",
+        "operationId": "getScreenshotByScanId",
```

Comment:
Please add x-ms-summary property

##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1beta~1scans~1{scanId}~1screenshot/get/parameters/0